tests/reftests/dune.inc

```diff
@@ -1,20 +1,20 @@
 
 (rule
- (alias reftest-admin-add-constraint)
```

Comment:
I think we can just keep `admin.test` and `admin-add-constraint` separate tests.
To me it makes more sense as `add-constraint` is the most used admin subcommand (plus it makes the change more readable)